content/docs/searching.md

```diff
@@ -271,7 +271,7 @@ More precisely:
 
 *Tip*:  It is possible to select and copy a rectangular selection of data from the **Search results** window.  This is done using the usual <kbd>Shift+Alt+arrow</kbd> keys or by holding <kbd>Alt+LeftClick</kbd> and dragging with the mouse.  This is really only practical when using the <kbd>Ctrl+C</kbd> method of copying; **<kbd>RightClick</kbd> > Copy Selected Line(s)** doesn't work this way.
 
-There is a capability to copy the list of files containing hits from past searches (v8.0.0 and later).  The **Copy Pathname(s)** context menu command copies to the clipboard the full pathnames of all files appearing in **Search results**.
+There is a capability to copy the list of files containing hits from past searches (v8.0.0 and later).  The **Copy Selected Pathname(s)** context menu command copies to the clipboard the full pathnames of all the files for any lines selected in in **Search results** (known as **Copy Pathname(s)** prior to v8.7.2).  Starting in v8.7.2, if you have selected search results, or if the [caret](#caret-and-cursor "typing/insertion cursor") is on just one of the search result entries, then this action will copy the pathname associated with each search result entry highlighted (or just the one at the caret); if you have no selection, but the selection [caret](#caret-and-cursor "typing/insertion cursor") is on one of the pathname headers, this action will copy that pathname.  Running this action on never-saved files, like the `new 1` tabs (even if they've been manually renamed but not yet saved), will use the tab's name as the "pathname", since there is no filesystem path associated with such tabs.
```

Comment:
This is more accurate and (very arguably!) reads better(?):

```
There is a capability to copy the list of files containing hits from past searches (v8.0.0 and later). The **Copy Selected Pathname(s)** context menu command (known as **Copy Pathname(s)** from v8.0.0 to v8.7.1) copies to the clipboard the full pathnames of all the files for any lines selected in **Search results**. The list copied to the clipboard will contain one line for each pathname. From v8.0.0 to v8.7.1, **Copy Pathname(s)** would copy all pathnames from all prior searches. Starting in v8.7.2, if you have selected search results that include a pathname line (even partially), or if the [caret](#caret-and-cursor "typing/insertion cursor") with no selected text is on one of the search result pathname lines, then this action will copy the pathname associated with each search result entry so selected. Running this action on never-saved files, like the `new 1` tabs (even if they've been manually renamed but not yet saved), will use the tab's name as the "pathname", since there is no filesystem path associated with such tabs. To mimic the behavior of **Copy Pathname(s)** from earlier software in v8.7.2 and later, select all text in search-results before running the action.
```

Take any or all...adjust as you will!  :-)

Reply:
Updated.